src/models/continuity/ContEven.ts

```diff
@@ -29,17 +29,32 @@ export default class ContEven
 
   getHumanReadableText(): string {
     if (this.humanReadableText !== "") return this.humanReadableText;
-    // TODO: Implement
-    return "";
+    if (this.marchType === MARCH_TYPES.HS) return "EVEN HS";
+    return "EVEN";
   }
 
-  /* eslint-disable @typescript-eslint/no-unused-vars */
   addToFlow(
     flow: FlowBeat[],
     startDot: StuntSheetDot,
     endDot?: StuntSheetDot
   ): void {
-    // TODO: Implement
+    if (endDot === undefined) return;
+    const [endX, endY]: [number, number] = [endDot.x, endDot.y];
+    const [startX, startY]: [number, number] = [startDot.x, startDot.y];
+    const [deltaX, deltaY]: [number, number] = [
+      (endX - startX) / this.duration,
+      (endY - startY) / this.duration,
+    ];
+    const dir =
+      (Math.atan2(deltaY, deltaX) * 180) / Math.PI + (deltaY < 0 ? 360 : 0);
+    for (let beat = 0; beat < this.duration; beat += 1) {
```

Comment:
Should this start at beat 1? I forgot to write this down, but the structure of `flow` is `[position at beat 1, position at beat 2]`, etc. So the position at beat 0 (the "Hup!" count) is handled by `dot.x` and `dot.y`. See the unit tests for dynamic eight to five to see what I mean
```suggestion
    for (let beat = 1; beat <= this.duration; beat++) {
```

Reply:
Wow, this actually turns out to be a really hard question.  Is there a 0-th beat?  If so, then does that mean that there is only a 0-th beat for stunt sheet 1?

Reply:
I think we discussed this on the slack channel, and determined that we should be counting the first beat of the continuity as the position but facing the new flow.  So I think what I have is correct?

Reply:
Lets leave this to be addressed with #100 Set the shows to have beats be [0, N)